src/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixer.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\LanguageConstruct;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\ConfigurationDefinitionFixerInterface;
+use PhpCsFixer\FixerConfiguration\AllowedValueSubset;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ */
+final class SingleSpaceAfterConstructFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
+{
+    /**
+     * @var array
+     */
+    private static $tokenMap = [
+        'break' => T_BREAK,
+        'continue' => T_CONTINUE,
+        'echo' => T_ECHO,
+        'goto' => T_GOTO,
+        'new' => T_NEW,
+        'print' => T_PRINT,
+        'return' => T_RETURN,
+        'throw' => T_THROW,
+        'yield' => T_YIELD,
+        'yield_from' => T_YIELD_FROM,
```

Comment:
Let me know if you have any ideas for constructs (and/or statements) I could add here!

Reply:
`require`, `include` and their `*_once` counterparts.

Reply:
Thank you, @derrabus!

Reply:
`clone`?

Reply:
@derrabus 

Implemented for

- `include`
- `include_once`
- `require`
- `require_once`

Reply:
@dmvdbrugge 

Implemented for

* `clone`